application-templates/starter-typescript/jest.test.config.js

```diff
@@ -0,0 +1,9 @@
+process.env.ENABLE_NEW_JSX_TRANSFORM = 'true';
+const jestPresetForTypeScript = require('@commercetools-frontend/jest-preset-mc-app/jest-preset-for-typescript');
```

Comment:
@emmenko wished to refactor this package and [the way we export the presets](https://github.com/commercetools/merchant-center-application-kit/tree/main/packages/jest-preset-mc-app#commercetools-frontendjest-preset-mc-app), but I guess I missed what is to be done here.
Right now there are separate presets for JS- (default) and TS-based (opt in) Custom Apps. Perhaps the point was to always support TS config out of the box? Perhaps it can also be done as a follow up?

Reply:
See #2627

Reply:
Got you, thanks 😊